.github/workflows/test.yaml

```diff
@@ -146,6 +146,14 @@ jobs:
         name: coverage
         path: coverage.enterprisepostgres.out
 
+    - name: upload diagnostics
+      if: ${{ always() }}
```

Comment:
I feel like I'm missing something with these if conditionals throughout :thinking:

Reply:
Normally, if an earlier step fails, Actions skips later steps We specifically want this to run when the earlier step (the test run) did fail.